tests/Zend/Locale/FormatTest.php

```diff
@@ -658,7 +658,7 @@ public function testIsDate()
 
         $this->assertFalse(Zend_Locale_Format::checkDateFormat('20.April', array('date_format' => 'dd.MMMM.YYYY')));
         $this->assertTrue(Zend_Locale_Format::checkDateFormat('20.April', array('date_format' => 'MMMM.YYYY'   )));
-        $this->assertTrue(Zend_Locale_Format::checkDateFormat('20', array('date_format' => 'dd.MMMM.YYYY')));
```

Comment:
These two changes are the incorrect tests. They shouldn't return true because the passed in date is missing segments defined in the format.